.circleci/config.yml

```diff
@@ -0,0 +1,323 @@
+
+  jest:
+    executor: client-executor
+    environment:
+      JEST_JUNIT_OUTPUT: coverage/junit.xml
+    steps:
+      - run:
+          command: apk add ca-certificates
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Jest
+          working_directory: client
+          command: yarn jest --ci --reporters=default --reporters=jest-junit --passWithNoTests
+      - store_test_results:
+          path: client/coverage
+
+  eslint:
+    executor: client-executor
+    steps:
+      - run:
+          command: apk add ca-certificates
+      - attach_workspace:
+          at: ./
+      - run:
+          name: ESLint
+          working_directory: client
+          command: yarn eslint src features
+
+  e2e:
+    executor: docker-executor
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - checkout
+      - run:
+          name: Start services
+          command: |
+            wget https://github.com/mikefarah/yq/releases/download/2.2.1/yq_linux_amd64
+            chmod +x ./yq_linux_amd64
+            docker-compose config --services | while read -r line ; do
+              ./yq_linux_amd64 delete -i docker-compose.yml "services.$line.volumes"
+            done
+            ./yq_linux_amd64 delete -i docker-compose.yml "volumes"
+            if [ ! -z $CI_SERVICE_ACCOUNT_KEY ]; then
+              echo -n $CI_SERVICE_ACCOUNT_KEY | base64 -d | docker login -u _json_key --password-stdin https://eu.gcr.io
+              source ci/.env
+              sed -i -E 's@CONTAINER_REGISTRY_BASE=.*@CONTAINER_REGISTRY_BASE='"$DOCKER_REPOSITORY"'@g' .env
+            fi
+            docker-compose pull --ignore-pull-failures
+            docker-compose build --pull
+            docker-compose up -d
+            sleep 10
+      - run:
+          name: Run tests
+          command: docker-compose exec -e REACT_APP_API_ENTRYPOINT=https://h2-proxy:8443 -e REACT_APP_SSE_ENTRYPOINT=https://h2-proxy:1338 client yarn e2e
+
+  # Deploy
+  deploy:
+    executor: deploy-executor
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - checkout
+      - restore_cache:
+          keys:
+            - vendor-{{ checksum "api/composer.lock" }}
+      - restore_cache:
+          keys:
+            - node_modules-{{ checksum "client/yarn.lock" }}
+      - deploy:
+          name: Deploy
+          command: |
+            source ci/.env
+            if [[ $REPOSITORY == $(echo $CIRCLE_REPOSITORY_URL | sed -E 's/^[^:]+:(.*)\.git/\1/g') && $CURRENT_CI == "circleci" ]] && [[ $FEATURE_DEPLOY == "1" || $DEPLOYMENT_BRANCH == $CIRCLE_BRANCH ]]; then
+              BRANCH=$CIRCLE_BRANCH
+              curl -sL https://deb.nodesource.com/setup_10.x | bash -
+              apt install nodejs
+              source ci/before_deploy
+              ci/deploy
+            else
+              echo "Deployment skipped"
+            fi
+
+workflows:
+  version: 2
+  test-and-deploy:
```

Comment:
Uhh, I've never realized I was doing the workflows incorrectly in api-platform/core :laughing:

I'll go fix that.

Reply:
I just checked your code. Didn't know orbs were available for free accounts, will try to implement them in this configuration